ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml

```diff
@@ -52,7 +52,7 @@ tests:
     test:
     - chain: openshift-qe-node-density-heavy
     workflow: openshift-qe-installer-azure-aro
-- as: ocp-qe-perfscale-ci-tests-6nodes-aro
+- as: ocp-qe-perfscale-cp-6nodes-aro
```

Comment:
@krishvoor Can we add periodic at the end of the name here?

Reply:
Sure, will do